src/templates/main/mixins/user_card_mixin.html

```diff
@@ -14,10 +14,12 @@
             </div>
             <!--Content-->
             <div class="card-block">
-                    <h4><a class="black-text" href="{{ userprofile.get_absolute_url }}">{{ userprofile.user.get_full_name|truncatewords:2 }}</a></h4>
-                <p>{{ designation|default:'' }}</p>
+                <h4><a class="black-text" href="{{ userprofile.get_absolute_url }}">{{ userprofile.user.get_full_name|truncatewords:2 }}</a></h4>
+                <p class = "h-48">{{ designation|default:''|truncatechars:75 }}</p>
```

Comment:
@ajatprabha maybe we can make an abbreviation field in societies and use that everywhere a card is used?
Or select first letter from each word, like CLS, STS etc